scripts/ps_dm_component.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+#check that data masking plugin is disabled
+DM_PLUGIN=$(mysql -NBe "SELECT PLUGIN_NAME, PLUGIN_STATUS FROM INFORMATION_SCHEMA.PLUGINS WHERE PLUGIN_NAME = 'data_masking';" | grep -c ACTIVE)
+
+if [ ${DM_PLUGIN} == 1 ]; then
+   echo "ERROR: DM plugin is installed and active"
+   exit 1
+else
+   echo "DM plugin isn't installed or active"
+fi
+
+#install the data masking component
+install=$(mysql -NBe "INSTALL COMPONENT 'file://component_masking_functions';")
```

Comment:
Why are we storing the execution result in install variable?
Why not simply execute the command

Reply:
I have removed `set -e` for 2 reasons:
1) If DM_PLUGIN had 0 result the whole script execution was failing
2) to run all the functions check and only then fail with the whole list of errors. 
Without `set -e` the script execution continues even if there are some errors in commands execution in script.

Reply:
Okey, got it now :) Will update the script.